lib/asciidoctor/converter/html5.rb

```diff
@@ -237,6 +211,41 @@ def document node
         end
         result << '</div>'
       end
+      
+      # Load Javascript at the end of body for performance
+      # See http://www.html5rocks.com/en/tutorials/speed/script-loading/
+      case node.attr 'source-highlighter'
+      when 'highlightjs', 'highlight.js'
+        highlightjs_path = node.attr 'highlightjsdir', %(#{cdn_base}/highlight.js/8.4)
+        result << %(<script src="#{highlightjs_path}/highlight.min.js"></script>
+<script>hljs.initHighlightingOnLoad()</script>)
+      when 'prettify'
+        prettify_path = node.attr 'prettifydir', %(#{cdn_base}/prettify/r298)
+        result << %(<script src="#{prettify_path}/prettify.min.js"></script>
+<script>document.addEventListener('DOMContentLoaded', prettyPrint)</script>)
```

Comment:
We can do the same here by calling `prettyPrint()` directly since the  DOM content is already loaded ?